environments/kubernetes/telemeter.libsonnet

```diff
@@ -30,4 +30,16 @@
       },
     },
   },
+} + {
+  local ts = super.telemeterServer,
+  telemeterServer:: {
+    [k]: ts[k]
+    for k in std.objectFields(ts)
+    // This array must be sorted for `std.setMember` to work.
+    if !std.setMember(k, [
+      'secret',
+      'serviceMonitor',
```

Comment:
why remove this serviceMonitor? We still want to scrape the telemeter servers' metrics no?

Reply:
Yes. But I talked to @aditya-konarde and we want to add these ServiceMonitors to their app-sre Prometheus. :blush:

Reply:
but we would still want the ServiceMonitor to be created in this namespace, no?

Reply:
It's a limitation of OpenShift 3.11 Prometheus Operator being on v0.22